packages/front-end/pages/datasources/[did].tsx

```diff
@@ -120,64 +97,51 @@ const DataSourcePage: FC = () => {
           <span className="badge badge-secondary">{d.type}</span>{" "}
           <span className="badge badge-success">connected</span>
         </div>
-        <div style={{ flex: 1 }} />
-        {canEdit && permissions.createDatasources && (
-          <div className="col-auto">
-            <DeleteButton
-              displayName={d.name}
-              className="font-weight-bold"
-              text="Delete"
-              onClick={async () => {
-                await apiCall(`/datasource/${d.id}`, {
-                  method: "DELETE",
-                });
-                mutateDefinitions({});
-                router.push("/datasources");
-              }}
-            />
-          </div>
-        )}
       </div>
 
       <div className="row">
         <div className="col-md-12">
           <div className="row mb-3">
             {canEdit && permissions.createDatasources && (
-              <div className="col-auto">
-                <a
-                  href="#"
-                  onClick={(e) => {
-                    e.preventDefault();
-                    setEditConn(true);
-                  }}
-                >
-                  <FaKey /> Edit Connection Info
-                </a>
-              </div>
-            )}
-            {(supportsSQL || supportsEvents) &&
-              canEdit &&
-              permissions.editDatasourceSettings && (
-                <div className="col-auto">
-                  <a
-                    href="#"
+              <div className="d-md-flex w-100 justify-content-between">
+                <div>
+                  <button
+                    className="btn btn-outline-primary mb-2 mb-md-0 mr-md-2 font-weight-bold"
                     onClick={(e) => {
                       e.preventDefault();
-                      setEditSettings(true);
+                      setEditConn(true);
                     }}
                   >
-                    <FaCode /> Edit Query Settings
```

Comment:
The "Edit Query Settings" text link was deleted since this screen has the editable sections.